site/en/docs/privacy-sandbox/attribution-reporting/define-filters/index.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Define custom rules using filters'
+subhead: >
+  Filters let you define rules for a variety of actions including which conversions are reported.
+description: >
+  Filters let you define rules for a variety of actions including which conversions are reported.
+date: 2022-03-01
+updated: 2023-03-14
+authors:
+  - maudn
+---
+
+Filters are custom fields you can create and set in your registration header. Filters are flexible and have more uses than we will cover here, but here are just two ways they can be useful.
+
+
+- Only count conversions for a specific product category, and filter out conversions for other categories.
+- Choose trigger data based on source event data: because the bit limit for the trigger data is different depending on the source type—ad click (0-7) or view (0 or 1)—it can be convenient to dynamically assign the value of the trigger data depending on the source type.
```

Comment:
```suggestion
- Choose trigger data based on source event data: because for event-level reports, the bit limit for the trigger data is different depending on the source type—ad click (0-7) or view (0 or 1)—it can be convenient to dynamically assign the value of the trigger data depending on the source type.
```